script.js

```diff
@@ -1,4 +1,81 @@
+
+// Define global variables
+var gamesPlayed = -1;
+var userWinCount = 0;
+var currentGameMode = 'waiting for user name'; 
+var computerObject = '';
+
+// Function to randomly generate a number which corresponds to either Scissors, Paper or Stone and return this object.
+function chooseRandomObject () {
+  var randomObject = Math.floor(Math.random()*3+1);
+  if (randomObject == 1) {
+    computerObject = 'Scissors';
+  }
+  else if (randomObject == 2) {
+    computerObject = 'Paper';
+  } else {
+    computerObject = 'Stone';
+  };
+  return computerObject;
+};
+
+
+// SPS logic 
+function playGame (userName, userGuess) {
+
+// Call the random object function to generate the computer guess
+var computerGuess = chooseRandomObject();
+var message = '';
+console.log(computerGuess,': computer guess');
+console.log(userGuess,': user guess')
+
+// Input validation to check that input is either Scissors, Paper or Stone. Must be spelled correctly and case sensitive.
+if (!(userGuess == 'Scissors' ||
+    userGuess == 'Paper' ||
+    userGuess == 'Stone')) {
+      // Display error message to user if input fails validation check
+      myOutputValue = `Oops, that doesn't seem right. <br> Please check your spelling and only select *one* object out of the following: <br> Scissors ✂️ Paper 📄 or Stone 💎`;
+      return myOutputValue;
+    }
+// Otherwise go ahead with the game
+  if (userGuess == computerGuess) {
+    message = `Unlucky ${userName}, that's a draw. <br> You guessed ${userGuess} and computer guessed ${computerGuess}.`;
+    } else if (
+      userGuess == 'Scissors' && computerGuess == 'Paper' ||
+      userGuess == 'Stone' && computerGuess == 'Scissors' ||
+      userGuess == 'Paper' && computerGuess == 'Stone'
+      ) {
+      // If user beats computer with any of the above 3 winning conditions, add 1 to the win count.
+        userWinCount += 1;
+        message = `Awesome ${userName}, you win! <br> You guessed ${userGuess} and computer guessed ${computerGuess}.`;
+      } else {
```

Comment:
Great use of `else` clause to simplify logic and remove need for complicated conditional statement